pkgs/games/vms-empire/default.nix

```
@@ -1,26 +1,55 @@
-{ lib, stdenv, fetchurl, ncurses, xmlto }:
+{ lib
+, stdenv
+, fetchurl
+, ncurses
+, xmlto
+, docbook_xml_dtd_44
+, docbook_xsl
+}:
 
 stdenv.mkDerivation rec {
   pname = "vms-empire";
-  version = "1.15";
+  version = "1.16";
 
   src = fetchurl{
-    url = "http://www.catb.org/~esr/vms-empire/${pname}-${version}.tar.gz";
-    sha256 = "1vcpglkimcljb8s1dp6lzr5a0vbfxmh6xf37cmb8rf9wc3pghgn3";
+    url = "http://www.catb.org/~esr/${pname}/${pname}-${version}.tar.gz";
+    hash = "sha256-XETIbt/qVU+TpamPc2WQynqqUuZqkTUnItBprjg+gPk=";
   };
 
-  buildInputs =
-  [ ncurses xmlto ];
+  buildInputs = [
+    ncurses
+    xmlto
+    docbook_xml_dtd_44
+    docbook_xsl
+  ];
 
-  patchPhase = ''
-    sed -i -e 's|^install: empire\.6 uninstall|install: empire.6|' -e 's|usr/||g' Makefile
+  postBuild = ''
+    xmlto man vms-empire.xml
+    xmlto html-nochunks vms-empire.xml
   '';
 
-  hardeningDisable = [ "format" ];
+  installPhase = ''
+    runHook preInstall
+    install -d ${placeholder "out"}/bin \
+               ${placeholder "out"}/share/doc/${pname} \
+               ${placeholder "out"}/share/man/man6 \
+               ${placeholder "out"}/share/applications/ \
+               ${placeholder "out"}/share/icons/hicolor/48x48/apps/ \
+               ${placeholder "out"}/share/appdata
+
+    install vms-empire ${placeholder "out"}/bin/
+    install vms-empire.html ${placeholder "out"}/share/doc/${pname}/
+  	install -T empire.6 ${placeholder "out"}/share/man/man6/vms-empire.6
+  	install vms-empire.desktop ${placeholder "out"}/share/applications/
+  	install vms-empire.png ${placeholder "out"}/share/icons/hicolor/48x48/apps/
+  	install vms-empire.xml ${placeholder "out"}/share/appdata/
```

```suggestion
    install -T empire.6 ${placeholder "out"}/share/man/man6/vms-empire.6
    install vms-empire.desktop ${placeholder "out"}/share/applications/
    install vms-empire.png ${placeholder "out"}/share/icons/hicolor/48x48/apps/
    install vms-empire.xml ${placeholder "out"}/share/appdata/
```

Also please install man pages with installManPage.